packages/builder/src/pages/builder/app/[application]/design/[screenId]/components/[componentId]/_components/settings/ConditionalUIDrawer.svelte

```diff
@@ -112,7 +112,7 @@
       Constants.OperatorOptions.NotEmpty.value,
     ]
     condition.noValue = noValueOptions.includes(newOperator)
-    if (condition.noValue) {
+    if (condition.noValue || condition.operator === "oneOf") {
```

Comment:
While `condition.operator` might always be up to date here (I'm not sure whether svelte will update a `bind:value` before calling an `on:change` or not) it would be safer to instead check `newOperator === "oneOf"`.

Reply:
Good catch! Using newOperator.